include/wx/platform.h

```diff
@@ -461,6 +461,15 @@
 #        ifndef MAC_OS_VERSION_11_0
 #           define MAC_OS_VERSION_11_0 110000
 #        endif
+#        ifndef MAC_OS_VERSION_12_0
+#           define MAC_OS_VERSION_12_0 120000
+#        endif
+#        ifndef MAC_OS_VERSION_11_0
```

Comment:
copy-paste error? Should be 13.

Reply:
Oops, yes, of course, thanks for noticing!